src/core/reference/include/ngraph/runtime/reference/interpolate.hpp

```diff
@@ -78,7 +78,8 @@ class GetNearestPixel final {
                     return static_cast<int64_t>(x_original);
                 }
             };
-        default:;
+        default: {
```

Comment:
Why do you need this change?

Reply:
This was caught by cpplint, so I changed.